src/api/.rubocop_todo.yml

```diff
@@ -1377,6 +1378,7 @@ Style/OptionalBooleanParameter:
     - 'lib/xpath_engine.rb'
 
 # Offense count: 54
+# Cop supports --auto-correct.
```

Comment:
Trying to fix these offenses with `--auto-correct` resulted in hitting this issue: https://github.com/rubocop-hq/rubocop/issues/9160

Therefore, we don't correct these offenses with `--auto-correct`, yet.